lib/dark_worlds_server_web/controllers/bot_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule DarkWorldsServerWeb.BotController do
+  use DarkWorldsServerWeb, :controller
+
+  alias DarkWorldsServer.Bot.BotClientSupervisor
+
+  plug :check_bot_server_enabled
+
+  def create(conn, %{"game_id" => game_id, "bot_count" => bot_count, "config" => config}) do
```

Comment:
I'm not quite sure how to test this, can I get some help?
For example, what is the config parameter, can it be an empty object?
Game id is an existing game?
For example, should this curl work?

```sh
curl
     --request POST 
    --url http://localhost:4000/ 
    --header 'content-type: application/json' 
    --data '{ game_id: 1, bot_count: 1, config: 1  }'
```